CONTRIBUTING.md

```diff
@@ -36,57 +36,57 @@ from the community if you ask your questions on
+  weeks of inactivity.
 
-*   Maintain clean commit history and use meaningful commit messages. PRs with
-    messy commit history are difficult to review and won't be merged. Use
-    `rebase -i upstream/master` to curate your commit history and/or to bring in
-    latest changes from master but avoid rebasing in the middle of a code
-    review.
+- Maintain clean commit history and use meaningful commit messages. PRs with
+  messy commit history are difficult to review and won't be merged. Use
+  `rebase -i upstream/master` to curate your commit history and/or to bring in
+  latest changes from master but avoid rebasing in the middle of a code
+  review.
 
-*   Keep your PR up to date with upstream/master. If there are merge conflicts,
-    we can't really merge your change.
+- Keep your PR up to date with upstream/master. If there are merge conflicts,
+  we can't really merge your change.
 
-*   All tests need to be passing before your change can be merged. We recommend
-    you run tests locally before creating your PR to catch breakages early on.
-    To test and check your commits locally, run the following:
+- All tests need to be passing before your change can be merged. We recommend
+  you run tests locally before creating your PR to catch breakages early on.
+  To test and check your commits locally, run the following:
 
-    -   `go test ./...` to run all tests
-    -   `go test -race ./...` to run tests in race mode
-    -   `go vet ./...` to find any functional issues
-    -   `golint ./...` to find any style issues (after installing [golint])
+  - `go test ./...` to run all tests
+  - `go test -race ./...` to run tests in race mode
+  - `go vet ./...` to find any functional issues
+  - `golint ./...` to find any style issues (after installing [golint])
 
-*   Go code should comply with the [Effective Go] guide and try to avoid
-    pitfalls in Go's [Code Review Comments]. It should be checked using [go vet]
-    and formatted using [gofmt]. Comments should adhere to the
-    [Documenting Go Code] post.
+- Go code should comply with the [Effective Go] guide and try to avoid
+  pitfalls in Go's [Code Review Comments]. It should be checked using [go vet]
+  and formatted using [gofmt]. Comments should adhere to the
+  [Documenting Go Code] post.
 
-*   Exceptions to the rules can be made if there's a compelling reason for doing
-    so.
+- Exceptions to the rules can be made if there's a compelling reason for doing
+  so.
 
 [golint]: https://github.com/golang/lint
-[Effective Go]: https://golang.org/doc/effective_go.html
-[Code Review Comments]: https://github.com/golang/go/wiki/CodeReviewComments
+[effective go]: https://golang.org/doc/effective_go.html
+[code review comments]: https://github.com/golang/go/wiki/CodeReviewComments
 [go vet]: https://golang.org/cmd/vet/
 [gofmt]: https://blog.golang.org/gofmt
-[Documenting Go Code]: https://blog.golang.org/godoc
+[documenting go code]: https://blog.golang.org/godoc
```

Comment:
I've never seen the capitalization of the reference not match its usage in markdown. For example, `[documenting go code]` is the reference defined but it is used as `[Documenting Go Code]`, since it is proper to capitalize articles. It does not appear to be a problem with GitHub Flavored Markdown rendering, but I thought I'd ask if this is desirable? As I said, I have not seen this before.

Reply:
Prettier does that by itself. I think it is desirable, because you may want to capitalize it in one place and not in another when referencing it, but not when defining it. So you might say `The purpose is to replace the [legacy framework]` but in a list you might say `-  [Legacy framework]`, with both referencing the same target defined in the footer.